server/utils/utils.ts

```diff
@@ -88,6 +88,23 @@ export const retrieveQuestionResponseFromApplication = <T>(
   }
 }
 
+export const retrieveOptionalQuestionResponseFromApplication = <T>(
+  application: ApprovedPremisesApplication,
+  task: string,
+  page: string,
+  question?: string,
```

Comment:
I solved this problem a different way on my branch. Wasn't sure whether to use the same function with options or a new one but I think it's better this way

Reply:
Yeah, I thought about adding options too, but I think it's clearer/simpler this way